sources/packages/backend/apps/db-migrations/src/migrations/1766516651982-AddChangeIntensityCOEDeniedReason.ts

```diff
@@ -0,0 +1,22 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { getSQLFileData } from "../utilities/sqlLoader";
+
+export class AddChangeIntensityCOEDeniedReason1766516651982 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      getSQLFileData(
+        "Add-change-intensity-coe-denied-reason.sql",
+        "COEDeniedReason",
+      ),
+    );
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      getSQLFileData(
+        "Rollback-add-change-intensity-coe-denied-reason.sql",
+        "COEDeniedReason",
+      ),
+    );
+  }
+}
```

Comment:
I would suggest adding the new column and inserting new value in a separate migration with their pertaining revert scripts.

Reply:
What would be the benefit of it?
As a pattern to be followed, unless the scripts belong to different tables/scopes, we should do it in a single transaction to avoid creating multiple files and migrations.